.vscode/cspell.global.yaml

```diff
@@ -96,6 +96,7 @@ ignoreWords:
     - mockexec
     - mockhttp
     - MYDIR
+    - nazd
```

Comment:
Use this PR instead: https://github.com/Azure/azure-dev/pull/4191

`nazd` should not be here. It is an issue of cspell not recognizing string scape `\n`

Reply:
Thx, i removed